androidshared/build.gradle.kts

```diff
@@ -65,6 +65,7 @@ dependencies {
     testImplementation(Dependencies.robolectric)
     testImplementation(Dependencies.mockito_kotlin)
     testImplementation(Dependencies.androidx_arch_core_testing)
+    testImplementation(Dependencies.androidx_fragment_testing)
```

Comment:
`fragmenttesting` is meant to prevent us from needing this (as it exposes it with `debugApi`). Is there definitely not a way to have that come in transitively?

If not, this should `debugImplementation` I believe (https://developer.android.com/guide/fragments/test).

Reply:
Ahh my mistake this was broken because I converted  `debugApi` to `debugImplementation` in the `fragmenttest` module. Fixed.